pom.xml

```diff
@@ -279,6 +279,12 @@
         <artifactId>gson</artifactId>
         <version>2.8.0</version>
     </dependency>
+
+    <dependency>
```

Comment:
Does this help? https://gist.github.com/criztovyl/358267e8375916ef6a8da404a3da5845